### ISSUE TITLE
'rethinkdb index-rebuild' does not work properly on Ubuntu

### ISSUE THREAD

**Issue by jeffquach** [rethinkdb/rethinkdb#4699]
After updating to 2.1.1, this message appears in the web UI:

    This cluster contains outdated indexes that were created with a previous version of RethinKDB that                contained some bugs.  Use rethinkdb index-rebuild to apply the latest bug fixes and improvements.See      the troubleshooting pagefor more details.

And when I run `rethinkdb index-rebuild` on the command line this pops up:

```
Error when launching 'rethinkdb-index-rebuild': No such file or directory
The rethinkdb-index-rebuild command depends on the RethinkDB Python driver, which must be installed.
If the Python driver is already installed, make sure that the PATH environment variable
includes the location of the backup scripts, and that the current user has permission to
access and run the scripts.
Instructions for installing the RethinkDB Python driver are available here:
http://www.rethinkdb.com/docs/install-drivers/python/
```

I don't understand why I would have to install the Python driver when I'm using the `rethinkdb` command line utility unless this is a bug.

This is my OS information when running `lsb_release -a`:

```
Distributor ID:	Ubuntu
Description:	Ubuntu 14.04.3 LTS
Release:	14.04
Codename:	trusty
```

This is the output from running `rethinkdb -v`:
```
rethinkdb 2.1.1~0trusty (GCC 4.8.2)
```

Am I encountering a bug or am I simply doing something wrong?

**Comment by Timer** [comment]
`rethinkdb index-rebuild` simply uses `execvp` to execute a python script which rebuilds the indices.
This script is shipped with the python driver, as seen [here](https://github.com/rethinkdb/rethinkdb/blob/next/drivers/python/rethinkdb/_index_rebuild.py). Thus why you need the python driver installed. It is simply more efficient to write administration tools using drivers instead of directly in the source code to keep a clean and maintainable code climate.

If you don't want to take my word for it ...
First look [here](https://github.com/rethinkdb/rethinkdb/blob/next/src/main.cc#L47), next [here](https://github.com/rethinkdb/rethinkdb/blob/next/src/clustering/administration/main/command_line.cc#L1626-L1627) and [here](https://github.com/rethinkdb/rethinkdb/blob/next/src/clustering/administration/main/command_line.cc#L51) -- these call [this](https://github.com/rethinkdb/rethinkdb/blob/next/src/clustering/administration/main/command_line.cc#L1590) which was registered [here](https://github.com/rethinkdb/rethinkdb/blob/next//drivers/python/setup.py#L24).

tl;dr it's not a bug, its the intended functionality -- install the python driver or migrate manually using [this as a reference](https://github.com/rethinkdb/rethinkdb/blob/next/drivers/python/rethinkdb/_index_rebuild.py) if you are against it.

**Comment by danielmewes** [comment]
It's like @Timer explained.
This issue is also relevant: https://github.com/rethinkdb/rethinkdb/issues/2080
